frontends/web/src/components/icon/logo.tsx

```diff
@@ -67,15 +67,15 @@ export const SwissMadeOpenSource = (props: GenericProps) => <img {...props} drag
 export const BitBoxBaseLogo = (props: GenericProps) => <img {...props} draggable={false} src={BaseLogo} alt="BitBoxBase" className={style.logo} />;
 
 const logoMap = {
-    'btc': [BTC, BTC_GREY],
-    'tbtc': [BTC, BTC_GREY],
-    'rbtc': [BTC, BTC_GREY],
-    'ltc': [LTC, LTC_GREY],
-    'tltc': [LTC, LTC_GREY],
-    'eth': [ETH, ETH_GREY],
-    'teth': [ETH, ETH_GREY],
-    'reth': [ETH, ETH_GREY],
-    'erc20Test': [ETH, ETH_GREY],
+    btc: [BTC, BTC_GREY],
+    tbtc: [BTC, BTC_GREY],
+    rbtc: [BTC, BTC_GREY],
+    ltc: [LTC, LTC_GREY],
+    tltc: [LTC, LTC_GREY],
+    eth: [ETH, ETH_GREY],
+    teth: [ETH, ETH_GREY],
+    reth: [ETH, ETH_GREY],
+    erc20Test: [ETH, ETH_GREY],
 
     'eth-erc20-usdt': [USDT, USDT_GREY],
```

Comment:
imho that is a bit of a useless lint, especially seeing that it leads mixed-style objects.

if you don't care about it please remove that lint, but no big deal either way :)

Reply:
How about "consistent-as-needed" https://eslint.org/docs/rules/quote-props#consistent-as-needed ?